vsts/vsts.yaml

```diff
@@ -1,460 +1,405 @@
-          COMMIT_FROM: $(COMMIT_FROM)
-
-      - script: |
-          rem Run dotnet first experience.
-          dotnet new
-
-          rem Start build
-          build.cmd -clean -build -configuration Debug -package
+  - task: securedevelopmentteam.vss-secure-development-tools.build-task-vulnerabilityassessment.VulnerabilityAssessment@0
+    displayName: 'Run Vulnerability Assessment'
+
+  - task: securedevelopmentteam.vss-secure-development-tools.build-task-publishsecurityanalysislogs.PublishSecurityAnalysisLogs@2
+    displayName: 'Publish Security Analysis Logs'
 
-        displayName: build
+  - task: securedevelopmentteam.vss-secure-development-tools.build-task-report.SdtReport@1
+    displayName: 'Create Security Analysis Report'
+    inputs:
+      AllTools: true
 
-      - task: ComponentGovernanceComponentDetection@0
-        displayName: 'Component Detection'
+  - task: securedevelopmentteam.vss-secure-development-tools.build-task-uploadtotsa.TSAUpload@1
+    displayName: 'TSA upload'
+    inputs:
+      tsaVersion: TsaV2
+      tsaStamp: Azure
+      codeBaseName: 'Azure-Iot-SDK-CSharp-Master'
 
-      - task: securedevelopmentteam.vss-secure-development-tools.build-task-antimalware.AntiMalware@3
-        displayName: 'Run MpCmdRun.exe'
-        inputs:
-          EnableServices: true
-          SignatureFreshness: OneDay
+  - task: securedevelopmentteam.vss-secure-development-tools.build-task-postanalysis.PostAnalysis@1
+    displayName: 'Post Analysis'
+    inputs:
+      AllTools: true
 
-      - task: securedevelopmentteam.vss-secure-development-tools.build-task-autoapplicability.AutoApplicability@1
-        displayName: 'Run AutoApplicability'
-        inputs:
-          ExternalRelease: true
+    # TODO #181 Enable post analysis to break builds after all above items are enabled.
+    enabled: false
 
-          IsSoftware: true
+# END: SDL and Compliance tasks #
 
-          UsesHSM: true
+- job: NETMF
+  displayName: NetMF
 
-      - task: securedevelopmentteam.vss-secure-development-tools.build-task-binskim.BinSkim@3
-        displayName: 'Run BinSkim'
-        inputs:
-          arguments: 'analyze  $(Build.SourcesDirectory)\Microsoft.Azure.Devices.*.dll $(Build.SourcesDirectory)\DeviceExplorer.exe --recurse --verbose'
+  condition: succeeded()
+  pool:
+    name: win
+    timeoutInMinutes: 1
+  steps:
+  - script: |
+       call "C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\Common7\\Tools\\VsDevCmd.bat"
+       vsts/windows_csharp_mf.cmd
 
-        # TODO #181 Config issue: must run on Debug builds only with valid PDBs.
-        enabled: false
+    displayName: build
 
-      - task: securedevelopmentteam.vss-secure-development-tools.build-task-codemetrics.CodeMetrics@1
-        displayName: 'Run CodeMetrics'
-        inputs:
-          Files: '$(Build.SourcesDirectory)\**\DeviceExplorer.exe;$(Build.SourcesDirectory)\**\Microsoft.Azure.Devices.*.dll'
+- job: DEVEXP
+  displayName: Device Explorer
+  steps:
```

Comment:
We should retain the timeout property; unless there's a reason not to

Reply:
I'll add it back in, sure